### PR TITLE
XW-87 | Display button only when it's possiblie to run modal

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: extensions/wikia/CuratedContent/CuratedContentHooks.class.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ private static function isAllowedWikia() {
 		);
 	}
 
-	private static function shouldDisplayCuratedContentToolButton() {
+	public static function shouldDisplayCuratedContentToolButton() {
```
Comment (bognix) [inline extensions/wikia/CuratedContent/CuratedContentHooks.class.php]: is hook a good place for this method? given we have CuratadContentHelper which is a bag of stuff maybe creating another one here is not the best idea

File: skins/oasis/modules/PageHeaderController.class.php [hunk 2]
```diff
@@ -153,8 +153,7 @@ protected function getDropdownActions() {
 	private function getCuratedContentButton() {
 		global $wgUser, $wgEnableCuratedContentExt;
 
-		if ( !empty( $wgEnableCuratedContentExt ) && WikiaPageType::isMainPage() &&
-			$wgUser->isAllowed( 'curatedcontent' ) ) {
+		if ( !empty( $wgEnableCuratedContentExt ) && CuratedContentHooks::shouldDisplayCuratedContentToolButton() ) {
```
Comment (bognix) [inline skins/oasis/modules/PageHeaderController.class.php]: maybe this check should happen inside CuratedContent::editButton?

**Comment by bognix** [comment]
LGTM

**Comment by vforge** [comment]
tested

**Pull request merged**